src/doc/rustc/src/lints/levels.md

```diff
@@ -330,4 +330,74 @@ $
 
 This feature is used heavily by Cargo; it will pass `--cap-lints allow` when
 compiling your dependencies, so that if they have any warnings, they do not
-pollute the output of your build.
+pollute the output of your build. However, note that `--cap-lints allow` does **not** override lints marked as `force-warn`.
+
+## Priority of Lint Level Sources
+
+Rust allows setting lint levels (`allow`, `warn`, `deny`, etc.) through various sources:
+
+- Attributes (`#[allow(...)]`, `#![deny(...)]`, etc.)
+- Command-line options (e.g., `--cap-lints`, `-A unused_variables`)
+
+When multiple lint levels apply to the same lint, the compiler resolves conflicts based on the **source’s scope and strength**.
```

Comment:
Nit: this wording is IMO a bit confusing, this is moreso about lint level *precedence*, there are no "conflicts", and I don't think "strength" is a thing.